inc/ThirdParty/Plugins/Ecommerce/ThemifyWooCommerceProductFilter.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace WP_Rocket\ThirdParty\Plugins\Ecommerce;
+
+use WP_Rocket\Event_Management\Subscriber_Interface;
+
+/**
+ * Subscriber for compatibility with Themify WooCommerce Product Filter.
+ *
+ * @since  3.11.0.5
```

Comment:
@jeawhanlee If you can change version to 3.11.3

Reply:
@CrochetFeve0251 Right 👌